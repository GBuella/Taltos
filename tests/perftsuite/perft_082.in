setboard 8/8/7k/7p/7P/7K/8/8 b - - 0 1
echo 8/8/7k/7p/7P/7K/8/8 b - - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
