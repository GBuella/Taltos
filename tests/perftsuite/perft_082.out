8/8/7k/7p/7P/7K/8/8 b - - 0 1
3
9
57
360
1969
10724
